src/utility/spi_drv.h

```diff
@@ -38,6 +38,10 @@
 	SpiDrv::waitForSlaveReady();  \
 	SpiDrv::spiSlaveSelect();
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
+static bool initialized = false;
+#pragma GCC diagnostic pop
 static bool initialized = false;
```

Comment:
This will still need to be removed though, otherwise you have a double declaration ;)